packages/common/src/authorization/projectMemberAbility.ts

```diff
@@ -91,34 +91,49 @@ export const projectMemberAbilities: Record<
                 },
             },
         });
-    },
-    editor(member, { can }) {
-        projectMemberAbilities.interactive_viewer(member, { can });
-        can('manage', 'SavedChart', {
-            projectUuid: member.projectUuid,
-            isPrivate: false,
-        });
         can('manage', 'Dashboard', {
             projectUuid: member.projectUuid,
-            isPrivate: false,
+            access: {
+                $elemMatch: {
+                    userUuid: member.userUuid,
+                    role: SpaceMemberRole.ADMIN,
+                },
+            },
         });
-        // should not check space access when creating a space
-        can('create', 'Space', {
+        can('manage', 'SavedChart', {
             projectUuid: member.projectUuid,
+            access: {
+                $elemMatch: {
+                    userUuid: member.userUuid,
+                    role: SpaceMemberRole.ADMIN,
+                },
+            },
         });
-        can('manage', 'Space', {
+        // should not check space access & role when creating a space
+        // allow interactive to create a space
+        can('create', 'Space', {
```

Comment:
Create space should be in editor. Interactive user should not be able to create new spaces.